stonesoup/updater/kernel.py

```diff
@@ -0,0 +1,109 @@
+from functools import lru_cache
+
+import numpy as np
+from scipy.stats import multivariate_normal
+
+from . import Updater
+from ..kernel import QuadraticKernel, Kernel
+from ..types.array import StateVectors
+from ..types.prediction import MeasurementPrediction
+from ..types.update import Update
+from ..base import Property
+
+
+class AdaptiveKernelKalmanUpdater(Updater):
+    """Adaptive kernel Kalman updater class
```

Comment:
would be useful to expand on this docstring to be consistent with documentation for other updaters.